src/components/forms/NetworkingRulesOrderCard.js

```diff
@@ -180,6 +189,10 @@ class NetworkingRulesOrderCard extends Component {
           name="Network Rules"
           showIfEmpty={this.state.rules}
         />
+        <OrderCardDataTable
```

Comment:
instead of the IcseHeading for Rules and ACLs, do we want to use the toolbar for the datatable? https://react.carbondesignsystem.com/?path=/story/components-datatable-toolbar--default

Reply:
thats a good idea ya